app-connector-client/server/app.ts

```diff
@@ -1,41 +1,39 @@
 #!/usr/bin/env node
 
-var express = require("express")
-
+import * as express from "express"
```

Comment:
Why here an asterisk is used while for the Logger it is not used? What is the difference between that imports?